text/0061-error-format-and-definitions.md

```diff
@@ -520,24 +494,124 @@ The maximum number of fields for a document is `65,535`. When this number is exc
 
 ---
 
-## invalid_ranking_rule
+## invalid_settings_displayed_attributes
 
-`Asynchronous`
+`Synchronous`
+
+### Context
+
+This error occurs when the user provided displayed attributes settings is invalid regarding its expected format.
+
+### Error Definition
+
+HTTP Code: `400 Bad Request`
+
+```json
+{
+    "message": "//TODO",
```

Comment:
@irevoire For now, I have marked a` //TODO` on every `message` field for newly introduced code replacing the generic bad_request one; We will update it to push the chosen format (a new one or re-use the serde one) :rage2: